samples/framework/libraries/migrate-library-csproj-keep-existing/tests/Car.Tests/Car.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <AssemblyName>Car.Tests</AssemblyName>
+    <PackageId>Car.Tests</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <RuntimeFrameworkVersion>1.0.4</RuntimeFrameworkVersion>
```

Comment:
I'd go with 1.1 here, and I think you only need the `TargetFramework` to make this work ...
```xml
<PropertyGroup>
  <TargetFramework>netcoreapp1.1</TargetFramework>
</PropertyGroup>
```
... everything else can (probably) go.